.github/workflows/platformDeploy.yml

```diff
@@ -374,9 +374,13 @@ jobs:
   postSlackMessageOnSuccess:
     name: Post a Slack message when all platforms deploy successfully
     runs-on: ubuntu-latest
-    if: ${{ success() }}
```

Comment:
Why did you remove this?

Reply:
Oh I saw that multiple messages were posted [here](https://expensify.slack.com/archives/C07J32337/p1646957073891919) and thought that it was also a problem because of [this](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds). 

I took another look and haven't seen multiple success messages being posted again, so it doesn't seem to be a problem after all. 

Updating the code to add the conditional back!